### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by AntoineGa**
fix typo merticName to metricName

### CODE REVIEW

**Comment by jeffhollan** [comment]
Thanks @AntoineGa!

File: pkg/scalers/kafka_scaler.go [hunk 1]
```diff
@@ -208,7 +208,7 @@ func (s *kafkaScaler) GetMetricSpecForScaling() []v2beta1.MetricSpec {
 }
 
 //GetMetrics returns value for a supported metric and an error if there is a problem getting the metric
-func (s *kafkaScaler) GetMetrics(ctx context.Context, merticName string, metricSelector labels.Selector) ([]external_metrics.ExternalMetricValue, error) {
+func (s *kafkaScaler) GetMetrics(ctx context.Context, metricName string, metricSelector labels.Selector) ([]external_metrics.ExternalMetricValue, error) {
```
Comment (jeffhollan) [inline pkg/scalers/kafka_scaler.go]: Line 236 uses this variable so you'd need to fix there too

Comment (AntoineGa) [reply pkg/scalers/kafka_scaler.go]: sorry, i have seen it during tgik and quickly try to fix it !

Comment (jeffhollan) [reply pkg/scalers/kafka_scaler.go]: No worries! Thanks for the help

**Pull request merged**